.appveyor.yml

```diff
@@ -0,0 +1,52 @@
+# DO NOT CHANGE the "init" and "install" sections below
```

Comment:
The main purpose of this commit was to 'implement mutate.R and setLabels.R'.  The files `.appveyor.yml` and `.codecov.yml` also appear in this commit for the first time, but they have nothing to do directly with mutate and setLabels. They should have been commited in a separate commit that addresses the continuous integration. To keep the version history clean, I encourage you to carefully check what files and code chunks you stage in git, and keep your commits more or less atomic and independent from each other :)

Reply:
This is just a general comment, no action required here.